packages/client/src/clients/in-app-messages/message-client.ts

```diff
@@ -57,7 +54,6 @@ export class InAppMessagesClient {
   > {
     const params = {
       ...this.defaultOptions,
-      ...options,
```

Comment:
Btw one thing I realized is if we are keeping the query key format as is (i.e. request route + params) then we would want to prune out the __loadingType and __fetchSource params. Again not blocking, I know I suggested we revisit this in a separate PR.